app/views/games/index.html.erb

```diff
@@ -26,8 +26,22 @@
       <div class="small-4 large-centered columns">
         <h3>Join one of these games</h3>
       </div>
-        <p> (list of all games. Have to add button to take user to that game still)
-        <h1><%= @games.inspect %></h1>
+
+
+        <div class="row">
+          <div class="small-4 small-centered columns">
+            <ul>
+              <% @games.each do |game| %>
+                <li>
+                  <%= game.name %>
+                  <%= link_to 'Join Game', game_path(game), :class => 'custom-button-class' %>
```

Comment:
This will link to the games#show action, right? Is there also code to actually join the game?